dockerfile/azure-rocm-nhc.dockerfile

```diff
@@ -103,6 +105,18 @@ RUN  mkdir -p /tmp/perftest && \
     cp COPYING ${AZ_NHC_ROOT}/LICENSES/perftest_LICENSE && \
     rm -rf /tmp/perftest
 
+# Update cmake
```

Comment:
I have expressed concern to Ravi about the way that we build Docker containers based on external container registries because we actually do not know exactly what they contain. In similar vein of "software supply chain" I wonder why we take CMake update direct from GitHub? Quesitons: do we actually need to update CMake? If yes, can we instead update just that package from Ubuntu packages vs GitHub?

Reply:
The new version of rccl, requires cmake 3.25 or newer and ubuntu 22.04 packages come with 3.22. We can compile and install cmake from source if it is better.